packages/terra-table/src/Table.jsx

```diff
@@ -243,9 +245,9 @@ function Table(props) {
   // -------------------------------------
   // event handlers
 
-  const handleColumnSelect = useCallback((columnId) => {
+  const handleColumnSelect = useCallback((columnId, cellCoordinates) => {
     if (onColumnSelect) {
-      onColumnSelect(columnId);
+      onColumnSelect(columnId, cellCoordinates);
```

Comment:
I would not expect consumers to get a cell coordinates argument in their onColumnSelect callback.